README.md

```diff
@@ -0,0 +1,207 @@
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Project Name] <a name="about-project"></a>
+
+This is my Microverse **Garden and Flower Show** site, created with HTML, Tailwind CSS, Javascript and mobile first approach.
+
+As my fist Capstone Project ending Module 1, I created a site to showcase all that I have learnt while completing the Portal.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>Tailwind CSS</summary>
+</details>
+
+<details>
+<summary>Javascript</summary>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Responsive**
+- **GitHub WorkFlow**
+- **Grid and Flexbox**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://hajnaloltyan.github.io/garden-and-flower-show/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+  - Code editor
+  - Node
+
+
+### Setup
+
+Clone this repository to your desired folder by opening the terminal and running the following command:
+
+```
+git clone git@github.com:hajnaloltyan/garden-and-flower-show.git
+
+```
+
+
+### Install
+
+Install this project with:
+
+```
+npm install
+
+```
+
+Then follow instructions from here to [install Tailwind CSS](https://tailwindcss.com/docs/installation)
+
+### Usage
+
+To run the project:
+
+  Open the Project in your code editor and start Live Server inside the index.html file. 
+
+### Run tests
+
+To run tests, run the following commands:
+
+  For HTML linter errors:
+
+```
+npx hint .
+
+```
+  For CSS linter errors:
+
+```
+npx stylelint "**/*.{css,scss}"
+
+```
+
+For Javascript linter errors:
+
+```
+npx eslint .
+
+```
+
+### Deployment
+
+Deployed by GitHub Pages.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Hajnalka Oltyan**
+
+- GitHub: [@hajnaloltyan](https://github.com/hajnaloltyan)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/hajnalka-oltyan/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
```

Comment:
Nice job on the README 📖 but as requested by the previous reviewer make sure to add some `Future Features` you're looking to implement later, since this project requires you to respect some color guidelines and you want to do otherwise, how about you add this to the `Future Features` to add the colors you want later, treat it as a todo-list. 😉